src/iOS.Core/Views/FormEntryTableViewCell.cs

```diff
@@ -195,5 +165,75 @@ public void Select()
                 TextField.BecomeFirstResponder();
             }
         }
+
+        public void BuildSecureCell(UIButton button)
+        {
+            button.TitleLabel.Font = UIFont.FromName("bwi-font", 28f);
+            button.SetTitle(BitwardenIcons.Eye, UIControlState.Normal);
+            button.TouchUpInside += (sender, e) =>
+            {
+                TextField.SecureTextEntry = !TextField.SecureTextEntry;
+                button.SetTitle(TextField.SecureTextEntry ? BitwardenIcons.Eye : BitwardenIcons.EyeSlash, UIControlState.Normal);
+            };
+        }
+
+        private void AddButtons(ButtonsConfig buttonsConfig)
+        {
+            Button = new UIButton(UIButtonType.System);
+            Button.Frame = ContentView.Bounds;
```

Comment:
You shouldn't need to set a `Frame` when you use constraints, except on certain occasions. Were you getting some kind of error without it?